pdf-view-restore.el

```diff
@@ -42,6 +42,16 @@
   :group 'pdf-view
   :type 'string)
 
+(defcustom use-file-base-name-flag "t"
```

Comment:
The "t" is a string, not boolean.

Reply:
Changed it to be just t. Resolving this.